src/webgpu/shader/execution/expression/unary/arithmetic.spec.ts

```diff
@@ -36,6 +41,21 @@ Accuracy: Correctly rounded
     u.combine('inputSource', allInputSources).combine('vectorize', [undefined, 2, 3, 4] as const)
   )
   .fn(async t => {
-    const cases = await d.get('negation');
+    const cases = await d.get('negation_f32');
     await run(t, unary('-'), [TypeF32], TypeF32, t.params, cases);
   });
+
+g.test('negation_i32')
```

Comment:
When I made this change, I decided to put the i32 and f32 tests in the same file, thinking it would be easier to add the other types as well (f16, u32, AFloat, AInt). However, after looking at other tests, I'm not sure this is a good idea. So if you want, I'll update this CL by splitting out the i32 tests into its own file.

Reply:
I would prefer to split this up

Reply:
Done. PTAL.